src/consensus.rs

```diff
@@ -55,15 +55,11 @@ use votor::Votor;
 pub const SLOTS_PER_WINDOW: u64 = 4;
 /// Number of slots in each epoch.
 pub const SLOTS_PER_EPOCH: u64 = 18_000;
-/// Time bound assumed on network transmission delays during periods of synchrony.
-const DELTA: Duration = Duration::from_millis(400);
```

Comment:
Rather than deleting these constants, I would actually use them, e.g.:
`const DELTA_EARLY_TIMEOUT: Duration = DELTA.checked_mul(2).unwrap();`
and
`const DELTA_TIMEOUT: Duration = DELTA_EARLY_TIMEOUT.checked_add(DELTA_BLOCK).unwrap();`

To actually use `DELTA_TIMEOUT`, we could then also change `votor.rs` from having:
`tokio::time::sleep(DELTA_BLOCK)).await;`
to
`tokio::time::sleep(DELTA_TIMEOUT.saturating_sub(DELTA_EARLY_TIMEOUT)).await;`

Reply:
Thanks, now updated.